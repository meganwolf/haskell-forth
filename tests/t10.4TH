2 DUP * .
10 DUP * CR .
10 STR DUP DUP CONCAT3 CR .
420 DUP - CR .
1337 DUP / 580 500 - * CR . 
5 DUP 3 - ^ CR .