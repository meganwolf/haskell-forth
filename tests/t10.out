4
100
101010
0
80
25